its/constant.ts

```diff
@@ -1,15 +1,34 @@
-export const SONARCLOUD_ORGANIZATION = "sonarsource-integration-tests";
+// Information about the Azure DevOps location where to run the E2E tests
+export const AZDO_BASE_URL = "https://dev.azure.com/";
+export const AZDO_ORGANIZATION = "sonarsource";
+export const AZDO_PROJECT = "SonarScannerAzdo";
+
+// Default branch name on GitHub
+export const GITHUB_MAIN_BRANCH = "master";
+
+// Information to bind to the SonarCloud used in the E2E tests
+export const SONARCLOUD_ORGANIZATION_KEY = "sonarsource-sonar-scanner-azdo-it";
 export const SONARCLOUD_SERVICE_CONNECTION = "SonarCloud";
 export const SONARQUBE_SERVICE_CONNECTION = "SonarQube";
 
-export const PROJECT_KEY = "azdo-it-project";
-export const PROJECT_NAME = "AZDO IT Project";
-
 export const CLI_VERSION_INPUT_NAME = "cliScannerVersion";
 export const DOTNET_VERSION_INPUT_NAME = "msBuildVersion";
 
-export const DUMMY_PROJECT_CLI_PATH = "dummy-project-cli";
-export const DUMMY_PROJECT_DOTNET_CORE_PATH = "dummy-project-dotnet-core";
-export const DUMMY_PROJECT_DOTNET_FRAMEWORK_PATH = "dummy-project-dotnet-framework";
-export const DUMMY_PROJECT_GRADLE_PATH = "dummy-project-gradle";
-export const DUMMY_PROJECT_MAVEN_PATH = "dummy-project-maven";
+export const DUMMY_PROJECT_CLI_KEY = "sonarsource-sonar-scanner-azdo-it_dummy-project-cli";
```

Comment:
A comment relating these constants to where they are triggered from

https://dev.azure.com/sonarsource/SonarScannerAzdo/_build

from what I understand, these pipelines will always exist here, and when we want to run them we manually trigger them; and when we want to add or remove them, we do it first from code, and then use the build task to generate a new yml config, and then from the link above, create a new pipeline using the new config

I like this! I'm sure you are already considering docs; I'm thinking out loud to ensure the flow is clear to me 👍🏻

Reply:
yes and yes, I'm adding comments now thanks for the callout 👍